src/main/resources/GuiConfigLayout.fxml

```diff
@@ -103,9 +103,9 @@
                     text="Select"/>
                 </HBox>
                 <HBox alignment="CENTER_LEFT" spacing="4.0">
-                  <TextField prefHeight="25.0" prefWidth="160.0" promptText="First Vote Col Index *"
+                  <TextField prefHeight="25.0" prefWidth="160.0" promptText="First Vote Col Index"
```

Comment:
Please update the descriptions in the txt file if you're going to change the "required" behavior here.